File: java/dsframework/src/main/java/dsf/route/RateLimitedException.java
package dsf.route;

/**
 *
 * @author arksea
 */
public class RateLimitedException extends RuntimeException {

    public RateLimitedException(String msg) {
        super(msg);
    }

    public RateLimitedException(String msg, Throwable ex) {
        super(msg, ex);
    }
}

File: csharp/dsf/dsf/src/route/RateLimitedException.cs
using System;

namespace com.baidu.dsf.route
{

	/// 
	/// <summary>
	/// @author arksea
	/// </summary>
	public class RateLimitedException : Exception
	{

		public RateLimitedException(string msg) : base(msg)
		{
		}

		public RateLimitedException(string msg, Exception ex) : base(msg, ex)
		{
		}
	}

}